ubcpi/static/js/src/d3-pibar.js

```diff
@@ -10,12 +10,14 @@ d3.custom = (d3.custom || {});
  *     {frequency: 0,  label: 'Option 5', class: 'ubcpibar'},
  * ]
  */
-d3.custom.barChart = function() {
+d3.custom.barChart = function(scope) {
     // Private Variables
     var chartWidth  = 750;
     var chartHeight = 250;
     var minTotalFrequency = 10;
 
+    if(scope.role == 'instructor' || scope.role == 'staff'){ minTotalFrequency = 1};
```

Comment:
In ubcpi-barchart-directive.js, this

role: '='

assigns the "role" attribute in ubcpi.html to the "role" scope property.

You can see the "role" attribute at the snippet

role="user_role">

in [ubcpi.html](https://github.com/ubc/ubcpi/blob/1f45d28508c6019fffd9af3857876ce32f09d4b2/ubcpi/static/html/ubcpi.html).